### PR TITLE
add support for jmp_buf

### PR DESCRIPTION
**Pull request by carnaval**
is it the right place for this to go ?

### CODE REVIEW

File: src/x86_64/registers.jl [hunk 1]
```diff
@@ -80,4 +80,29 @@ module X86_64
     return RC
   end
 
+  function glibc_unmangle_ptr(x)
```
Comment (Keno) [inline src/x86_64/registers.jl]: Why is this needed? Needs a comment to explain.

**Comment by carnaval** [comment]
ups forgot about that, it's not very useful for now since it needs a julia patch to access the jmp_buf of a Task anyway, i'll make a PR again when it has landed
